docs/developersetup.md

```diff
@@ -0,0 +1,43 @@
+# Developer Setup
+---
+This page contains information about how to set up your local environment to develop for this project. 
+
+## Overview of Stack
+---
+Homebrew: This is a package manager that lets us install other packages.
+Docker: This will load all the dependencies you need in an “image” so you don’t mess up your local computer. We launch a container and do all our hacking in there.
+Visual Studio Code: This is an IDE that has git/terminal/docker integration. You’ll write your code here
+
+## macOS Setup
+---
+
+1. Install Homebrew with: /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+2. Install Docker with: brew install docker
+3. Install Git with: brew install git
+4. Install IDE
+
+## Git 
+---
+
+### Setting up your local repo
+
+1. Clone repo locally with: git clone https://github.com/mellychat/webapp.git 
```

Comment:
You can use code blocks to make the document cleaner. For example,

```
1. Clone repo locally with:
`git clone ...`
```

The backticks (`) will put it on it's own line. Ditto for the other lines (use a code block for emphasis).